cmd/controller/app/options/insecure_serving.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2024 The Godel Scheduler Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/spf13/pflag"
+
+	cmdconfig "github.com/kubewharf/godel-scheduler/cmd/controller/app/config"
+	godelctrlmgrconfig "github.com/kubewharf/godel-scheduler/pkg/controller/apis/config"
+	apiserveroptions "k8s.io/apiserver/pkg/server/options"
+)
+
+// CombinedInsecureServingOptions sets up to two insecure listeners for healthz and metrics. The flags
+// override the ComponentConfig and DeprecatedInsecureServingOptions values for both.
+type CombinedInsecureServingOptions struct {
+	Healthz *apiserveroptions.DeprecatedInsecureServingOptionsWithLoopback
+	Metrics *apiserveroptions.DeprecatedInsecureServingOptionsWithLoopback
+
+	BindPort    int    // overrides the structs above on ApplyTo, ignored on ApplyToFromLoadedConfig
+	BindAddress string // overrides the structs above on ApplyTo, ignored on ApplyToFromLoadedConfig
+}
+
+// AddFlags adds flags for the insecure serving options.
+func (o *CombinedInsecureServingOptions) AddFlags(fs *pflag.FlagSet) {
+	if o == nil {
+		return
+	}
+
+	fs.StringVar(&o.BindAddress, "address", o.BindAddress, "DEPRECATED: the IP address on which to listen for the --port port (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces). See --bind-address instead.")
```

Comment:
Better to remove those deprecated flags.

Reply:
I think it is best to keep the behavior consistent with other scheduler components, retain the insecure server configuration, and clean it up later.